src/containers/Tenant/Query/ExplainResult/ExplainResult.tsx

```diff
@@ -153,43 +150,21 @@ export function ExplainResult({
 
     return (
         <React.Fragment>
-            <div className={b('controls')}>
-                {!loading && (
-                    <React.Fragment>
-                        <div className={b('controls-right')}>
-                            <QueryExecutionStatus error={error} />
-                            {!error && (
-                                <React.Fragment>
-                                    <Divider />
-                                    <RadioButton
-                                        options={explainOptions}
-                                        value={activeOption}
-                                        onUpdate={(tabId) => {
-                                            startTransition(() => setActiveOption(tabId));
-                                        }}
-                                    />
-                                </React.Fragment>
-                            )}
-                        </div>
-                        <div className={b('controls-left')}>
-                            {copyText && (
-                                <ClipboardButton
-                                    text={copyText}
-                                    view="flat-secondary"
-                                    title={i18n('action.copy', {activeOption})}
-                                />
-                            )}
-                            <EnableFullscreenButton disabled={Boolean(error)} />
-                            <PaneVisibilityToggleButtons
-                                onCollapse={onCollapseResults}
-                                onExpand={onExpandResults}
-                                isCollapsed={isResultsCollapsed}
-                                initialDirection="bottom"
-                            />
-                        </div>
-                    </React.Fragment>
-                )}
-            </div>
+            <ResultControls<QueryExplainTab>
```

Comment:
It look like there is no much sense in splitting explain and execute results to different components now. Probably, it better merge them into one component, addded an issue for this: https://github.com/ydb-platform/ydb-embedded-ui/issues/1120